packages/terra-utils/lib/index.ts

```diff
@@ -0,0 +1,94 @@
+import { Block, SwapParams, Transaction, terra } from '@liquality/types'
+import { addressToString, validateExpiration, validateSecretHash, validateValue } from '@liquality/utils'
+import { InvalidAddressError } from '@liquality/errors'
+import { DateTime } from 'luxon'
+
+export const normalizeBlock = (data: any): Block => ({
+  hash: data.block_id.hash,
+  timestamp: convertDateToTimestamp(data.block.header.time),
+  size: Number(data.block.header.height),
+  number: Number(data.block.header.height),
+  parentHash: data.block.last_commit.block_id.hash
+})
+
+export const normalizeTransaction = (
+  data: any,
```

Comment:
Where complicated one 🗡️  Yes they do have TxInfo but then most of the values becomes red like they are not recognized/not part of the object which is kinda weird because I extract the from it